widget/ui/src/icons/Rango.tsx

```diff
@@ -0,0 +1,31 @@
+import type { SvgIconPropsWithChildren } from '../components/SvgIcon';
```

Comment:
Please move & rename this component to `/components/Logo`. The content for this directory should be autogenerated only.

Reply:
Remove the svg file from resources as well.